src/java.desktop/share/legal/lcms.md

```diff
@@ -61,15 +61,15 @@ Robin Watts (Artifex)
 Shawn Pedersen
 Andrew Brygin
 Samuli Suominen
-Florian Hˆch
+Florian Höch
 Aurelien Jarno
 Claudiu Cebuc
 Michael Vhrel (Artifex)
 Michal Cihar
 Daniel Kaneider
 Mateusz Jurczyk (Google)
 Paul Miller
-SÈbastien LÈon
+Sébastien Léon
```

Comment:
I cannot comment on capitalization here, but if we wanted to lowercase them, should they be e-grave instead of e-acute?

Reply:
If this is a French name, it's e acute: é.

Reply:
It's not a mistake in capitalization, it's a mistake for two different characters in two different encodings. (Probably iso-8859-1 mistaken as ansi iirc.)

I verified the developers name at the original file in the LCMS repo.